weaviate/README.md

```diff
@@ -10,14 +10,58 @@ Follow the instructions below to install and configure this check for an Agent r
 
 ### Installation
 
-The Weaviate check is included in the [Datadog Agent][2] package.
-No additional installation is needed on your server.
+The Weaviate check is included in the [Datadog Agent][2] package. No additional installation is needed on your server.
 
-### Configuration
+**Note**: This check requires Agent v7.47.0+.
 
-1. Edit the `weaviate.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your weaviate performance data. See the [sample weaviate.d/conf.yaml][4] for all available configuration options.
+### Configuration
 
-2. [Restart the Agent][5].
+Weaviate can be configured to expose Prometheus-formatted metrics. The Datadog Agent can collect these metrics using the integration described below. Follow the instructions to configure data collection from any or all of the components. For the required configurations, please refer to the [Weaviate documentation][11].
+
+In addition, a small subset of metrics can be collected by communicating with different [API endpoints][12].
```

Comment:
Could you clarify `different API endpoints`? Based on the link provided, are these OpenMetrics endpoints?

Reply:
Got it, thanks for clarifying. What metrics would be available from these endpoints, and why would users want to collect them? Would it be possible to list the metrics, or maybe provide some examples?

Reply:
@buraizu I added a few lines in regards to each endpoint.